Java TZUpdater Sprint - WMS 11461

jasonbegy opened the pull request:
# WMS 11461

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [N] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

anooshapilli reviewed (commented):
@jasonbegy - Could you rename image titles to descriptive title and make PNG lower case
<img width="305" alt="image" src="https://github.com/oracle-livelabs/sprints/assets/63126232/aa61c16a-abc1-436f-8d5d-6e117932ca25">

jasonbegy commented:
I have updated the filenames per your request. I do not know why the extensions were capitalized since they were lower-case on my machine, but since I changed the filenames the issue seems to be resolved.
